pkg/chunk/cache/memcached.go

```diff
@@ -248,7 +248,7 @@ func (c *Memcached) Stop() error {
 // HashKey hashes key into something you can store in memcached.
 func HashKey(key string) string {
 	hasher := fnv.New64a()
-	hasher.Write([]byte(key)) // This'll never error.
+	_, _ = hasher.Write([]byte(key)) // This'll never error.
```

Comment:
Could add this to the project `.errcheck-exclude`

Reply:
@zendern May you do it, please?